jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/skaffold/FilesMojoV2.java

```diff
@@ -207,6 +216,8 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     }
 
     try {
+      System.out.println(
```

Comment:
Is this a debug print or is it actually needed?

Reply:
This should've broken the tests... weird.

Reply:
oh maven tests are different than gradle ones... nvm.